Fails on macos 10.13:

```
builder for '/nix/store/l9g39yk5rmqmpvwrp3pcc0342s8av45g-simpleitk-1.2.2.drv' failed with exit code 2; last 10 log lines:
  [ 97%] Swig source to generate TCL wrapping
  Scanning dependencies of target SimpleITKTclsh
  [ 97%] Building CXX object Wrapping/Tcl/CMakeFiles/SimpleITKTclsh.dir/SimpleITKTCL_wrap.cxx.o
  /tmp/nix-build-simpleitk-1.2.2.drv-0/source/build/Wrapping/Tcl/SimpleITKTCL_wrap.cxx:166:10: fatal error: 'tcl.h' file not found
  #include <tcl.h>
           ^~~~~~~
  1 error generated.
  make[2]: *** [Wrapping/Tcl/CMakeFiles/SimpleITKTclsh.dir/build.make:407: Wrapping/Tcl/CMakeFiles/SimpleITKTclsh.dir/SimpleITKTCL_wrap.cxx.o] Error 1
  make[1]: *** [CMakeFiles/Makefile2:2554: Wrapping/Tcl/CMakeFiles/SimpleITKTclsh.dir/all] Error 2
  make: *** [Makefile:152: all] Error 2
```

--- a/pkgs/development/libraries/simpleitk/default.nix
+++ b/pkgs/development/libraries/simpleitk/default.nix
@@ -1,13 +1,14 @@
-{ stdenv, fetchurl, cmake, git, swig, lua, itk }:
+{ stdenv, fetchFromGitHub, cmake, git, swig, lua, itk, tcl, tk }:
 
 stdenv.mkDerivation rec {
-  pname    = "simpleitk";
-  version = "1.2.0";
-  name  = "${pname}-${version}";
+  pname = "simpleitk";
+  version = "1.2.2";
 
-  src = fetchurl {
-    url    = "https://sourceforge.net/projects/${pname}/files/SimpleITK/${version}/Source/SimpleITK-${version}.tar.gz";
-    sha256 = "10lxsr0144li6bmfgs646cvczczqkgmvvs3ndds66q8lg9zwbnky";
+  src = fetchFromGitHub {
+    owner = "SimpleITK";
+    repo = "SimpleITK";
+    rev = "v${version}";
+    sha256 = "1cgq9cxxplv6bkm2zfvcc0lgyh5zw1hbry30k1429n9737wnadaw";
   };
 
   nativeBuildInputs = [ cmake git swig ];
@@ -21,7 +22,7 @@ stdenv.mkDerivation rec {
     homepage = http://www.simpleitk.org;
     description = "Simplified interface to ITK";
     maintainers = with maintainers; [ bcdarwin ];
-    platforms = platforms.unix;
+    platforms = platforms.linux;
     license = licenses.asl20;
   };
 }

